src/components/include/DefaultImportModalDialog.jsx

```diff
@@ -0,0 +1,320 @@
+import nativeFormats from "@exabyte-io/made.js/lib/parsers/native_formats";
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    FormControl,
+} from "@mui/material";
+import { DataGrid } from "@mui/x-data-grid";
```

Comment:
Same here if possible

Reply:
+1

Reply:
When I import as DataGrid then I get an Error:
'@mui/x-data-grid' does not contain a default export (imported as 'DataGrid').